Incoprporate feedback (#1)
NBISweden/module-BYOD-dm-practices: _episodes/05.md

---

> title: Create a data dictionary for my dataset

Reviewer: or maybe "Create a dictionary for one of my datasets"

> title: Create a data dictionary for my dataset

Author: ```suggestion
title: Create a dictionary for one of my datasets
```

## Create a data dictionary file 

> 1. Define a dataset you would like to create a data dictionary for.

Reviewer: Maybe: "Identify a dataset for which you want to create a data dictionary."

> 1. Define a dataset you would like to create a data dictionary for.

Author: ```suggestion
1. Identify a dataset for which you want to create a data dictionary.
```
